tasks/src/zero-shot-image-classification/about.md

```diff
@@ -8,4 +8,20 @@ The data in this learning paradigm consists of
 
 - Seen data - images and their corresponding labels
 - Unseen data -  only labels and no images
-- auxiliary information - additional information given to the model during training connecting the unseen and seen data. This can be in the form of textual description or word embeddings.
\ No newline at end of file
+- auxiliary information - additional information given to the model during training connecting the unseen and seen data. This can be in the form of textual description or word embeddings.
```

Comment:
```suggestion
- Auxiliary information - additional information given to the model during training connecting the unseen and seen data. This can be in the form of textual description or word embeddings.
```

Nit.

Reply:
However, the notion of seen and unseen is not clear to me from the context.